src/components/index/Header.js

```diff
@@ -106,12 +106,15 @@ const Header = () => (
                 </div>
                 <div className="header__banner">
                     <h2 style={{ fontSize: '2.1rem' }}>
-                        <a href="https://eclipsesource.com/blogs/2023/12/29/eclipse-theia-1-45-release-news-and-noteworthy/" rel="noopener noreferrer">Learn about the latest 1.45 Release</a> and <a href="https://eclipsesource.com/blogs/2023/12/14/the-eclipse-theia-community-release-2023-11/">the latest Community Release (2023-11)</a>.
+                        <a href="https://eclipsesource.com/blogs/2024/02/09/eclipse-theia-1-46-release-news-and-noteworthy/" rel="noopener noreferrer">Learn about the latest 1.46 Release</a> and <a href="https://eclipsesource.com/blogs/2024/02/08/the-eclipse-theia-community-release-2023-11/">the latest Community Release (2024-02)</a>.
```

Comment:
https://eclipsesource.com/blogs/2024/02/08/the-eclipse-theia-community-release-2023-11/ is the right link although 2024-02 would be expected. We should make sure that the url has the right name in it for future releases.